### DISCUSSION TITLE
Can I run sysbox in a already existing container that is not privileged?

### DISCUSSION THREAD

**Discussion by John-Ao** [nestybox/sysbox#880] [Q&A] [answered]
From the documents, it seems that sysbox need to be run on the host machine to provide docker-in-docker service. But I only have access to a CentOS 7 container (with root user) that is not privileged and I cannot access the host machine. Is it still possible to run sysbox in that container so that I can run another container (with Ubuntu 20.04)? The new container does not have to be able to run system-level softwares.

**Comment by ctalledo** [comment, marked as answer]
Hi @John-Ao, 

>  Is it still possible to run sysbox in that container so that I can run another container

Yes, as long as the container that Sysbox is running in is a privileged container (e.g., `docker run --privileged ...`). In addition, you also need to mount some host dirs into the container. 

In fact, to test Sysbox, we run it inside such a privileged container. You can see the `docker run` command we use to create the privileged container [here](https://github.com/nestybox/sysbox/blob/208470292bee3c4d8f38b0fd8872127ea5e5b741/Makefile#L302). You would need to do something similar. 

If the underlying container is not privileged, then there's currently no way to run Sysbox in it unfortunately.

Hope that helps!
